src/creator/tyler/Main.java

```diff
@@ -13,7 +13,7 @@
     public void start(Stage primaryStage) throws Exception
     {
         Parent root = FXMLLoader.load(getClass().getResource("sample.fxml"));
-        primaryStage.setTitle("Hello World");
+        primaryStage.setTitle("Calculator");
```

Comment:
Maybe we could call this "Super Calc" instead?!?  Marketing loves it!

Reply:
We'll stick with this title, thank you very much!